### PR TITLE
DS-1016 | Sync tour changes and clean up; update packages

### PR DESCRIPTION
**Pull request by yvonnetangsu**
# READY FOR REVIEW

# Summary
- Package update
- Event banner honor new superhead field added for Tour
- More robust check for background blur for various components. This is due to how the content author can manually delete an option from a dropdown field (single option), even though the "hide empty" option is selected in Storyblok.
- Add new quoteSource field for Featured Masonry component (it was hardcoded before - a mistake)

# Review By (Date)
- Soon or Retro

# Criticality
- 5

# Review Tasks

## Setup tasks and/or behavior to test

1. Go to https://deploy-preview-365--giving-campaign.netlify.app/stories/education-for-a-life-of-purpose and check that the quote in this component still has the source (I added the hardcoded quote source into Storyblok and published the story)
Screenshot 2024-10-21 at 11 37 13 AM
2. Go to https://deploy-preview-365--giving-campaign.netlify.app/experience-the-moment and check that all the Event banners have superheading and some of them are custom, e.g., "Haas Center"
2. Look at code

# Associated Issues and/or People
- JIRA ticket(s) - DS-1016

### CODE REVIEW

File: components/Storyblok/SbEventBanner.tsx [hunk 1]
```diff
@@ -68,6 +70,7 @@ export const SbEventBanner = ({
     <EventBanner
       {...storyblokEditable(blok)}
       id={id}
+      superhead={superhead || 'Event'}
```
Comment (yvonnetangsu) [inline components/Storyblok/SbEventBanner.tsx:73]: Passing a fallback superhead here. I found out that if we pass that in the variable higher up, sometimes Storyblok empty string can override the default.

File: components/FeaturedStories/FeatureMasonry.tsx [hunk 2]
```diff
@@ -80,7 +82,7 @@ export const FeatureMasonry = ({
               quoteColor="flamingo"
               body={quoteBody}
               isLargeBody
-              source="Eden Hadar, ’26"
```
Comment (yvonnetangsu) [inline components/FeaturedStories/FeatureMasonry.tsx:83]: They have only use this component on one story (the original plan was that it was just for this story). I made some attempt to make it available for general usage, but I missed this bit 🤦🏼‍♀️

File: components/Storyblok/SbSection/SbSection.tsx [hunk 3]
```diff
@@ -105,6 +105,7 @@ export const SbSection = ({
   const hasHeader = heading || superhead || subheading;
   // To render a dark overlay, both a top and bottom gradient color must be selected
   const hasBgGradient = !!gradientTop && !!gradientBottom;
+  const hasBgBlur = !!bgBlur && bgBlur !== 'none';
```
Comment (yvonnetangsu) [inline components/Storyblok/SbSection/SbSection.tsx:108]: More robust check for whether background blur was selected

**Comment by yvonnetangsu** [comment]
@sherakama The changes here should be pretty safe so I'm going to merge this into dev. I'm going to cut a release to get our recent changes in (including the CTA link updates) 🤠

**Review by sherakama — COMMENTED** [review]
Thanks

**Pull request merged**